test/router_test.rb

```diff
@@ -1,9 +1,12 @@
 require 'test_helper'
 require 'gds_api/router'
+require 'gds_api/test_helpers/router'
 
 describe GdsApi::Router do
+  include GdsApi::TestHelpers::Router
+
   before do
-    @base_api_url = "http://router-api.example.com"
+    @base_api_url = Plek.current.find('router-api')
```

Comment:
I've been on a 5 year quest to eradicate the use of `Plek.current` because it's entirely pointless.

Plek has a shortcut for `Plek.find` here:
https://github.com/alphagov/plek/blob/master/lib/plek.rb#L156